### ISSUE TITLE
dependencies

### ISSUE THREAD

**Issue by peterkasson** [aspuru-guzik-group/Pasithea#5]
Apologies if I'm missing it, but would it be possible to add a statement of dependencies?
From a quick look at the code, it looks like Selfies, Pytorch, Numpy, RDKit?

Thanks!

**Comment by MarioKrenn6240** [comment]
Yes, it requires SELFIES (this specific code was developed with [SELFIES v1.0.x](https://github.com/aspuru-guzik-group/selfies/releases), [higher version needs modification](https://github.com/aspuru-guzik-group/Pasithea/issues/4)), numpy, pytorch, pandas, RDKit.
